src/main/resources/application.yaml

```diff
@@ -0,0 +1,15 @@
+server:
+  port: 8080
+
+
+spring:
```

Comment:
additionally we would also like to return the database connectivity status in the response of the `/ping` endpoint.

Reply:
sorry, i guess the commits were in the main branch, seeing it now..

for the success / error response, please create two DTOs. 
example: https://github.com/pkonopko/ScheduleApp/blob/master/src/main/java/com/example/demo/models/webDTOs/TeacherWebDto.java

these models encapsulate the data that is being transferred between the controller and the api clients.

Reply:
I added DTO's and updated PingContoroller